go.mod

```diff
@@ -22,7 +22,9 @@ require (
 	github.com/json-iterator/go v1.1.12
 	github.com/onsi/ginkgo/v2 v2.14.0
 	github.com/onsi/gomega v1.30.0
-	github.com/openshift-kni/oran-o2ims/api/hardwaremanagement v0.0.0-00010101000000-000000000000
+	github.com/openshift-kni/oran-o2ims/api/hardwaremanagement v0.0.0-20241001130125-a052f08603f7
```

Comment:
Why is this version being changed?

Reply:
I don't think it should have been. Plus, we have that performed by ci-job on each PR. The update includes upversioning multiple dependencies, in addition to the restructuring. Maybe you inadvertently also upgraded this module?

Reply:
I noticed the hardware manager gets included as a required module with a version containing a string with today's date when I do `go mod tidy` for the provisioning package after the initial `go mod init` for provisioning. :thinking: 
Then after I do `go mod tidy` for the main oran-o2ims package, the main `go.mod` gets updated to the same hardware manager version.

Reply:
I think the only way this would have happened would be if you had run `go get github.com/openshift-kni/oran-o2ims/api/hardwaremanagement`. Ultimately, it doesn't really matter, since there's a `replace` directive. It's just an unnecessary change that sticks out and made me curious